### PR TITLE
WIP: DB-API EXCEPTIONS

### PR DESCRIPTION
**Pull request by mf2199**
This PR is to serve as a template for concurrent work on the Django-Spanner project.

Whether you are working on the core implementation, testing, docstrings or samples, please commit all you changes here, so that the others can see them too. This way multiple people can work on the same features with minimum conflicts.

### CODE REVIEW

File: google/cloud/spanner_dbapi/exceptions.py [hunk 1]
```diff
@@ -23,8 +23,7 @@ class Error(Exception):
 
 
 class InterfaceError(Error):
-    """
-    Error related to the database interface
+    """Error related to the database interface
```
Comment (IlyaFaer) [inline google/cloud/spanner_dbapi/exceptions.py:26]: We already had a conversation with Valentina about it yesterday.

PEP 257 says we can use both styles:

Безымянный

Googlers and their code generation bots are writing in both ways, and doesn't always write a summary line at all.

Comment (mf2199) [reply google/cloud/spanner_dbapi/exceptions.py:26]: That is correct. What I'd propose is consistency: we do it either but consistent way, including the common style of Spanner's main repo which normally has summaries placed on the same line with the opening quotes.

Comment (IlyaFaer) [reply google/cloud/spanner_dbapi/exceptions.py:26]: Oh, I get it. I'm driven by such a logic: if a summary line is one-line, than write it right after the `"""`; if a summary line is too long to stay within one line, then start it on the next line.

So, it is:
```python
"""One line summary.

Detailed description
"""
```

But:
```python
"""
Too long summary, long summary, long summary, long summary, long summary,
long summary, long summary, long summary, long summary, long summary.
"""
```
In fact, I can't remember where I saw this recommendation, probably in Google Napoleon style Tim Swast once gave me. 

What do you think?

**Comment by mf2199** [comment]
Closing this, as the Exceptions have already been implemented.

**Pull request closed without merging**